sql/src/main/java/io/crate/exceptions/RelationUnknown.java

```diff
@@ -27,17 +27,17 @@
 import java.util.Collections;
 import java.util.Locale;
 
-public class TableUnknownException extends ResourceUnknownException implements TableScopeException {
+public class RelationUnknown extends ResourceUnknownException implements TableScopeException {
```

Comment:
Why is the exception missing the `Exception` suffix, e.g. `RelationUnknownException`? AFAIK we always use this naming.